packages/vetrina/src/Vetrina/Purchase/NewPurchase/Order.purs

```diff
@@ -131,11 +131,11 @@ mkPurchase props logger window askAccount validForm affUser = do
         let pError = props.purchaseError (hush eitherUser)
         case err of
           EmailInUse email ->
-            pError (Just $ ExistingAccount email) validForm.productSelection NewPurchase
+            pError (Just $ ExistingAccount { email: email, invalidPassword: false }) validForm.productSelection NewPurchase
```

Comment:
I think this `email: email` is redundant.

Reply:
It's now fixed.